build-npm-module/index140.js

```diff
@@ -0,0 +1 @@
+// First create a `getUpperCase()` function which take a string variable `str` and return upper case of this string
```

Comment:
I think an example should be given so it is clear what format the function should be in. As you may know, when a user goes to the next step, the code shown will be the code we create in these challenges, not necessarily the code the user typed in. So we want to make sure the code the user types in is as close as possible to the 'correct' code.

Maybe something like:
// First create a `getUpperCase()` function which take a string variable `str` and return upper case of this string. For example, here is a `getTrimmed()` function which takes a string variable and returns the string with no whitespace on the end: `const getTrimmed = (str) => str.trim();`